Controller specs

gustavosobral opened the pull request.

gustavosobral committed:
Add StatsController specs

gustavosobral committed:
Fix error response on show action from CurriculaController

gustavosobral committed:
Add CurriculaController specs

gustavosobral committed:
Reset student sequence on Program Spec

gustavosobral committed:
Solve Rubocop detected offenses

fernandobrito commented on spec/controllers/stats_spec.rb:
```diff
@@ -0,0 +1,57 @@
+
+    it 'is programs average equals to 5 and 3' do
+      get :students
+      expect(assigns(:programs_average)).to include(@program_cc => 7.34)
```
Where does 7.34 comes from?

gustavosobral replied:
It comes from the default Student defined on Student Factory. Maybe I should explicit the Student average_grade value in this case...

gustavosobral committed:
Explicit average_grade value on stats spec

gustavosobral committed:
Fix typos on stats specs description

gustavosobral committed:
Update sample-transcript to fix sensitive information

gustavosobral committed:
Add binary files to fixtures spec folder

gustavosobral committed:
Override Rack UploadFile class

This way, is possible to keep the uploaded file extension when
transformed in tempfile

gustavosobral committed:
Add specs to TranscriptsController
